fares-data-build-tool/create-data
# feat: cfd 454 view point to point product details

# Description

-   View details for point to point products

# Testing instructions

-   Create one of each ticket listed in CFD-454 jira ticket and ensure correct fields are present for each

# Type of change
